venvUtils/ensureVenv.py

```diff
@@ -94,7 +94,7 @@ def populate():
 			"-m",
 			"pip",
 			"install",
-			"setuptools==70",
+			"setuptools==72.0.0",
```

Comment:
I think we want to match any 72 subversion 

```suggestion
			"setuptools~= 72",
```

Reply:
As requested previously, please fixup the code comment above this line which references py2exe and setuptools